migrations/2023-07-28-111829_update_columns_to_fix_db_diff/up.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE dispute
+ALTER COLUMN payment_id TYPE VARCHAR(64);
+
+ALTER TABLE merchant_key_store
+ALTER COLUMN created_at SET DEFAULT now();
```

Comment:
The defaults were intentionally removed. See #991 and #984. (Also, PR #1043 has a commit that removes these defaults.)

Reply:
Okay, removing this.